documentation/index.html

```diff
@@ -831,12 +835,117 @@ <h2 class="font-black my-2">Lists</h2>
+                                <li class="ml-4">
+                                    <a href="#" target="_blank"><i
+                                            class="fa fa-linkedin-square navbar-icons transition-2"></i></a>
+                                </li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <div class="my-3 snippet">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%28250%2C250%2C250%2C1%29&t=material&wt=sharp&l=htmlmixed&width=977&ds=false&dsyoff=20px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=136%25&si=false&es=2x&wm=false&code=%253Cnav%2520class%253D%2522navbar%2520fixed%2520left-0%2520top-0%2520z-1%2520flex-row%2520items-center%2520wrap%2520py-2%2520px-4%2520w-100%2522%253E%250A%2520%2520%2520%253C%21--%2520Your%2520logo%2520or%2520brand%2520goes%2520here%2520--%253E%250A%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522black%2522%253EHero%2520logo%253C%252Fh3%253E%250A%2520%2520%2520%2520%253C%252Fa%253E%250A%250A%2520%2520%2520%2520%253Cul%2520class%253D%2522ml-auto%2520flex-row%2520items-center%2520font-bold%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522ml-4%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520target%253D%2522_blank%2522%253E%253Ci%2520class%253D%2522fa%2520fa-github%2520navbar-icons%2520transition-2%2522%253E%253C%252Fi%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522ml-4%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520target%253D%2522_blank%2522%253E%253Ci%2520class%253D%2522fa%2520fa-twitter%2520navbar-icons%2520transition-2%2522%253E%253C%252Fi%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522ml-4%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520target%253D%2522_blank%2522%253E%253Ci%2520class%253D%2522fa%2520fa-linkedin-square%2520navbar-icons%2520transition-2%2522%253E%253C%252Fi%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%253C%252Ful%253E%250A%253C%252Fnav%253E%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509%2509"
+                            style="height: 419px; border:0; transform: scale(1); overflow:hidden;"

```

Comment:
Suggestion: You can remove the inline styles here.

Reply:
It's from carbon.sh, changing those styles changes the responsiveness, so haven't removed it.